elytron/src/main/java/org/wildfly/extension/elytron/CredentialStoreService.java

```diff
@@ -114,6 +114,9 @@ public void start(StartContext startContext) throws StartException {
             credentialStoreAttributes.put(CS_LOCATION_ATTRIBUTE, loc.toAbsolutePath().toString());
             credentialStore = getCredentialStoreInstance();
             credentialStore.initialize(credentialStoreAttributes, resolveCredentialStoreProtectionParameter(), otherProviders.getOptionalValue());
+            if ( credentialStoreAttributes.get(ElytronDescriptionConstants.CREATE).equals("true") && !loc.toFile().exists() ){
+                credentialStore.flush();
```

Comment:
[@darranl] [@yersan] I recall merging a recent change that had some catch logic related to flush, IIRC to ensure that the corresponding model change was reverted. Is that needed here?